src/test/logger.ts

```diff
@@ -0,0 +1,18 @@
+import { ActionLogger } from "../github/types";
+
+export class TestLogger implements ActionLogger {
+  logHistory: string[] = [];
```

Comment:
How is `logHistory` being used?

Reply:
It is not being used yet, but I'll eventually use it for validations.

Got that idea from `pr-custom-review`